### DISCUSSION TITLE
VS Code Extensions

### DISCUSSION THREAD

**Discussion by AzamBukhari** [AammarTufail/python_ka_chilla2023#1] [General]
Hello
While installing VS Code extensions on a PC/laptop. Do we need Admin rights on that machine as well?

I am using my work laptop, and not yet installed VS Code & Python. 
Wanted to clarify about extensions, so that I can ask the IT guy to install it while he is installing above two soft wares

Regards
Azam

**Comment by Izharhussain2022** [comment]
Ask for local admin rights (if you don't have one yet). I believe you don't have to rely on IT every time you need to install something. Once you have local admin rights, you can proceed with any installations. If there is any suspicious source, your company firewall will block it anyway!

**Reply by Masoodullah909** [reply]
Be cautious when installing software from unknown sources as it may contain malware or other security risks. Check with your IT department or do research online to verify that the software is legitimate. Your company's firewall should block any suspicious or malicious software, but avoid installing software from untrusted sources whenever possible.

To install software or make certain changes to the system, you will need local administrator rights. To request these rights, contact your IT department or the person responsible for managing user accounts on your network. Explain that you need the rights to install Visual Studio Code extensions or other necessary software.

**Reply by AzamBukhari** [reply]
@Izharhussain2022 , @Masoodullah909 AoAlaikum, JazakAllah Khair. All sorted. many thanks

**Comment by usamabajwa86** [comment]
Intelicode extension not found in VSCODE 
![image](https://user-images.githubusercontent.com/120989108/210306438-a416a5a9-260e-4f00-a160-78b75db30a20.png)

**Reply by Muhammad-Umar1995** [reply]
Intellicode

**Comment by baberibrar** [comment]
> Hello While installing VS Code extensions on a PC/laptop. Do we need Admin rights on that machine as well?
> 
> I am using my work laptop, and not yet installed VS Code & Python. Wanted to clarify about extensions, so that I can ask the IT guy to install it while he is installing above two soft wares
> 
> Regards Azam

In general, you do not need administrator rights to install Visual Studio Code or its extensions on a PC. However, you may need administrator rights to install certain extensions that require system-level access or make changes to system files.

**Comment by QuratUlAin946** [comment]
I have installed the Blackbox extension, but after installing this whenever I write "//How to make a line plot in python?" in VS code it does not show any code. Where is the mistake?

**Reply by Izharhussain2022** [reply]
It works for me
![image](https://user-images.githubusercontent.com/121819549/210345726-208ca712-11df-4a70-bfd6-eb8ce4bf3a7a.png)

Try using backspace or press enter

**Reply by zohaib-qureshi** [reply]
I used 'Tab' after the '?' and it worked

**Comment by Masoodullah909** [comment]
> Hello While installing VS Code extensions on a PC/laptop. Do we need Admin rights on that machine as well?
> 
> I am using my work laptop, and not yet installed VS Code & Python. Wanted to clarify about extensions, so that I can ask the IT guy to install it while he is installing above two soft wares
> 
> Regards Azam

Visual Studio Code Extensions are additional packages of code that enhance the functionality of the code editor. They can add new features, support new programming languages, and more. You can install extensions directly from within Visual Studio Code by going to the Extensions tab and searching for the extension you want to install. You can also browse for extensions on the Visual Studio Code Marketplace website.

To install Visual Studio Code extensions, you do not need administrator rights. You can install extensions to your personal user folder without any special permissions. However, if you are trying to install an extension system-wide, or if you are trying to install an extension to a location that you do not have permission to access, you will need administrator rights.

**Comment by khansamehek** [comment]
![20230105_233200](https://user-images.githubusercontent.com/121963691/210912072-6662cbc7-0ff9-433c-810c-b43a213fae66.jpg)


Ya issue ho rha hy now bi extension install ko ni hot I ya error a jata hy

**Reply by AzamBukhari** [reply]
Can you please share log

Also check disk space on ur system

**Reply by SahilJatoi744** [reply]
Ap apna internet or disk space dekh waha issue ho sakta hai
